### ISSUE TITLE
The object 'beyond' doesn't show in the 3d viewport, but they appear in the svg. 

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#4165]
In the following file, the object 'beyond' doesn't show in the 3d viewport, but they appear in the svg. 

file: https://hub.openingdesign.com/OpeningDesign/Monica_Residence/src/commit/d340027b35ee1c8719cf0278e789f2951d011015/Models/BlenderBIM/Monica_Residence.ifc

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/ba973728-2aa0-4f83-900d-f42dade63773)

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/507113/ff397260-db76-4c7b-a992-e2b1ec754363)

**Comment by aothms** [comment]
Yeah I think we never did anything with the far plane in the SVG serializer. Should indeed be fairly straight forward.

**Comment by Moult** [comment]
Hmm, that might be potentially more performant than booleans but I'm afraid we might run into the missing lines issue described here https://github.com/IfcOpenShell/IfcOpenShell/issues/3561 which @aothms would know the answer to.
